syncd/Makefile.am

```diff
@@ -20,6 +20,12 @@ if SAICAVM
 SAIFLAGS = -DCAVMSAI
 endif
 
+if SAIVS
+SAILIB=-L$(top_srcdir)/vslib/src/.libs -lsaivs
```

Comment:
>.libs [](start = 33, length = 5)

.libs is not necessary. This is an example: [https://github.com/Azure/sonic-swss-common/blob/master/tests/Makefile.am](https://github.com/Azure/sonic-swss-common/blob/master/tests/Makefile.am)